components/src/core/AppBar/AppBar.tsx

```diff
@@ -208,7 +207,7 @@ const AppBarRender: React.ForwardRefRenderFunction<unknown, AppBarProps> = (prop
         else if (previousOffset > 0 && offset === 0) {
             expandToolbar();
         }
-    }, [offset, scrollThreshold]);
+    }, [offset, scrollThreshold, animating, collapseToolbar, expandToolbar, previousOffset, variant]);
```

Comment:
Changing these dependencies means that this useEffect will run any time one of the dependencies changes. The description of the useEffect says it's intended to run only when the offset crosses the scrollThreshhold...has this change impacted functionality?